perf/istio-install/setup_istio.sh

```diff
@@ -26,23 +26,13 @@ mkdir -p "${WD}/tmp"
 
 release="${1:?"release"}"
 
-if [[ "${release}" == *-latest ]];then
```

Comment:
-dev is similarly resolved some place else. Can you check if it done in one of the other files?

Reply:
yes, it is resolved in other PR already, would update accordingly